src/dummy/toga_dummy/factory.py

```diff
@@ -52,6 +53,7 @@ def not_implemented(feature):
     'Button',
     'Canvas',
     'DetailedList',
+    'Divider',
```

Comment:
You've added the Divider to the *dummy* factory, but not the *cocoa* factory.

Reply:
Yes, I forgot to push the last commit.